acestep/llm_inference.py

```
@@ -113,6 +114,16 @@ def unload(self) -> None:
         except Exception:
             pass
 
+    def _cleanup_torch_distributed_state(self) -> None:
+        """Destroy default torch distributed process group when already initialized."""
+        try:
+            import torch.distributed as dist
+            if dist.is_available() and dist.is_initialized():
+                logger.warning("[LLM vLLM] Destroying stale default process group before/after vLLM lifecycle")
+                dist.destroy_process_group()
+        except Exception as exc:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)